### PR TITLE
93234 - VA Profile - Maintenance Windows

### PR DESCRIPTION
**Pull request by nickjg231**
## Summary
- Update maintenance window service keys. 
- Update existing VA Profile components using DowntimeNotification component with new service keys
- Create new DowntimeNotification components in VA Profile components where it was missing

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/93234

## Testing done
- Manual testing

## Screenshots

|                              | Normal                                                                                   | Window Approaching                                                                                 | Window Active                                                                                       |
| ---------------------------- | ---------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
| Profile                      | image | image | N/A                                                                                                 |
| Account Security              | image | image | image |
| Connected Apps                | image | image | image |
| Contact Information           | image | image | image |
| Direct Deposit Information    | image | image | image |
| Military Information          | image | image | image |
| Notification Settings         | image | image | image |
| Personal Health Care Contacts | image | image | image |
| Personal Information          | image | image | image |

## What areas of the site does it impact?
- Various VA Profile pages

## Acceptance criteria
- Maintenance windows work as expected on each page

### CODE REVIEW

**Review by asiisii — COMMENTED** [review]
It seems that the tests are still failing. Could you clarify why the render=`{handleDowntimeForSection('')}` prop was removed? Also, please add a comment explaining the use of the `UNKNOWN` value for `externalServices`.

**Comment by nickjg231** [comment]
> It seems that the tests are still failing. Could you clarify why the render=`{handleDowntimeForSection('')}` prop was removed? Also, please add a comment explaining the use of the `UNKNOWN` value for `externalServices`.

I removed the render function because the function `handleDowntimeForSection` is a custom function that will only render the maintenance window message when it's active, not when it's upcoming. When the component renders with its default rendering, rather than through a prop function, then it will show the upcoming and active states.

**Comment by tpharrison** [comment]
@asiisii @nickjg231 Instead of using "UNKNOWN," should we consider "NOT-NEEDED" (or something similar)? I know we agreed to "UNKNOWN," but now that I think about it more, it might imply that a service exists but we don't know what it is. "NOT-NEEDED" might better convey that the intent is not to tie this to a service.

**Comment by asiisii** [comment]
> @asiisii @nickjg231 Instead of using "UNKNOWN," should we consider "NOT-NEEDED" (or something similar)? I know we agreed to "UNKNOWN," but now that I think about it more, it might imply that a service exists but we don't know what it is. "NOT-NEEDED" might better convey that the intent is not to tie this to a service.

Thanks for the explanation  @nickjg231 

and yes @tpharrison , that's how I was reading it and why I requested for comments on that line. I like the `NOT-NEEDED` one much better

File: src/applications/personalization/profile/mocks/server.js [hunk 1]
```diff
@@ -146,7 +146,14 @@ const responses = {
...
     //   maintenanceWindows.createDowntimeActiveNotification([
-    //     maintenanceWindows.SERVICES.VA_PROFILE,
+    //     maintenanceWindows.SERVICES.VAPRO_PROFILE_PAGE,
+    //     maintenanceWindows.SERVICES.VAPRO_CONTACT_INFO,
+    //     maintenanceWindows.SERVICES.LIGHTHOUSE_DIRECT_DEPOSIT,
+    //     maintenanceWindows.SERVICES.VAPRO_MILITARY_INFO,
+    //     maintenanceWindows.SERVICES.VAPRO_NOTIFICATION_SETTINGS,
+    //     maintenanceWindows.SERVICES.VAPRO_HEALTH_CARE_CONTACTS,
+    //     maintenanceWindows.SERVICES.VAPRO_PERSONAL_INFO,
+    //     maintenanceWindows.SERVICES.UNKNOWN,
```
Comment (asiisii) [inline src/applications/personalization/profile/mocks/server.js:156]: do we still need this `UNKNOWN` service?

**Review by asiisii — APPROVED** [review]
LGTM 🚀

**Review by pjhill — APPROVED** [review]

**Pull request merged**